solutions/devicesimulation-nohub/single-vm/setupWrapper.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+# Note: Windows Bash doesn't support shebang extra params
+
+./setup.sh $@ > /dev/null 2>setup-errors.log
```

Comment:
This file should be named to setup.sh and the old setup.sh renamed to setupInternal.sh. Add a comment here as to why we are doing it this way to avoid secrets leaking out to the logs on azureiotsolutions.com

Reply:
good suggestion, updated. also added comment.